apps/products/models.py

```diff
@@ -0,0 +1,250 @@
+import os
+import uuid
+from django.db import models
+from apps.masters.models import *
+from django.dispatch import receiver
+from django.db.models.signals import pre_delete
+from utils_methods import EncryptedTextField
+from utils_variables import *
+
+
+def product_groups_picture(instance, filename):
+    # Get the file extension
```

Comment:
why this method is outside the class? if it is common method and can be reuesed then move to utils_methods.py file

Reply:
Actually from picture column we are calling that method Thulasi
    picture = models.ImageField(max_length=255, default=None, null=True, upload_to=product_groups_picture)
and we tried putting it in utils_methods but everyone having different requirements so we left like that only .